### DISCUSSION TITLE
Office_S_Tips/system/2020/02/20/Backup_dysku_SSD

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#28] [Komentarze]
# Office_S_Tips/system/2020/02/20/Backup_dysku_SSD

Przegląd kilku systemów backupu.

https://andrzejq.github.io/Office_S_Tips/system/2020/02/20/Backup_dysku_SSD.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
